src/components/Projects/index.js

```diff
@@ -27,6 +28,11 @@ import Chip from '@material-ui/core/Chip';
         slug
         bookmarked
         gitBranch
+        features(showDeployed: true, params: { limit: 25 }) {
```

Comment:
What happens if the number of new features exceeds 25?

Reply:
It should not matter anything more than one is already an alert to see whats up

Reply:
So could we just say "new features available" or something like that rather than give potential misinformation? I'm fine merging for now, but just something to think about